pkg/objstore/azure/helpers.go

```diff
@@ -18,7 +19,8 @@ import (
 	"github.com/Azure/go-autorest/autorest/azure/auth"
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
-	"github.com/thanos-io/thanos/pkg/objstore"
+	"github.com/thanos-io/thanos/pkg/exthttp"
+	//"github.com/thanos-io/thanos/pkg/objstore"
```

Comment:
Please avoid commenting code in a PR (: It's easy to merge such and create a bit of mess 🙈